1.8.1: Unable to build tarball if git not present

**Bug Description:**
The [custom targets in CMakeLists.txt](https://github.com/surge-synthesizer/surge/blob/9de93366e343e45e0da427b00a944d24d5b38304/CMakeLists.txt#L1484-L1492) prevent CMake from generating Makefiles when using the release source tarball.

**Surge Version**
n/a

* Version: 1.8.1
* Plugin Type: n/a
* Bitness: 64-bit

**Reproduction Steps:**
Steps to reproduce the behavior:

```
  cmake -DCMAKE_INSTALL_PREFIX='/usr' \
        -DCMAKE_BUILD_TYPE='None' \
        -W no-dev \
        -B build \
        -S .
```

```
-- The CXX compiler identification is GNU 10.2.0
-- The ASM compiler identification is GNU
-- Found assembler: /usr/bin/c++
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- CMake Version is 3.19.6
-- Compiler Version is 10.2.0
-- Building in 64 bit configuration
-- Could NOT find Git (missing: GIT_EXECUTABLE)
-- Performing Test SURGE_FILESYSTEM_STD_FS_FOUND
-- Performing Test SURGE_FILESYSTEM_STD_FS_FOUND - Success
-- filesystem: Using std::filesystem from <filesystem>
-- The C compiler identification is GNU 10.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Could NOT find Git (missing: GIT_EXECUTABLE)
-- VERSION_GIT_INFO file is present; using that rather than git query
-- Setting up surge version
--   git hash is 9de9336 and branch is release/1.8.1
--   buildhost is dave
--   buildarch is x86_64
-- Developer Local Build
-- Configuring a Release Build from 'release/1.8.1'
-- Using SURGE_VERSION=1.8.1.9de9336
-- Configuring /build/surge/src/surge-1.8.1/build/geninclude/version.cpp
-- Found PkgConfig: /usr/bin/pkg-config (found version "1.7.3")
-- Checking for module 'cairo'
--   Found cairo, version 1.17.4
-- Checking for module 'fontconfig'
--   Found fontconfig, version 2.13.91
-- Checking for module 'x11'
--   Found x11, version 1.7.0
-- Checking for module 'xcb'
--   Found xcb, version 1.14
-- Checking for module 'xcb-cursor'
--   Found xcb-cursor, version 0.1.3
-- Checking for module 'xcb-keysyms'
--   Found xcb-keysyms, version 0.4.0
-- Checking for module 'xcb-util'
--   Found xcb-util, version 0.4.0
-- Checking for module 'xcb-xkb'
--   Found xcb-xkb, version 1.14
-- Checking for module 'xkbcommon'
--   Found xkbcommon, version 1.0.3
-- Checking for module 'xkbcommon-x11'
--   Found xkbcommon-x11, version 1.0.3
-- Looking for C++ include pthread.h
-- Looking for C++ include pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
-- Python bindings disabled
-- Configuring done
CMake Error at CMakeLists.txt:1492 (add_dependencies):
  The dependency target "download-extra-content" of target
  "code-quality-pipeline-checks" does not exist.


-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
```

**Expected Behavior:**
CMake is able to generate a Makefile from the sources in the release tarball.

**Screenshots:**
n/a

**Computer Information (please complete the following information):**
 - OS: Arch Linux
 - Host: n/a
 - Version: n/a

**Additional Information:**
Custom targets, that are only relevant for CI/CD should be hidden behind an option, that is off by default and is only used in CI.

--- a/cmake/stage-extra-content.cmake
+++ b/cmake/stage-extra-content.cmake
@@ -28,4 +28,16 @@ if( ${Git_FOUND} )
             WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
             COMMAND cmake -E copy_directory ${CMAKE_BINARY_DIR}/surge-extra-content/Skins/ resources/data/skins
             )
+else()
+    message(STATUS "Absent git, making extra content commands no-ops")
+    add_custom_target(download-extra-content)
+    add_custom_command(TARGET download-extra-content
+            POST_BUILD
+            WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
+            COMMAND cmake -E echo Skipping Extra Content)
+    add_custom_target(stage-extra-content)
+    add_custom_command(TARGET stage-extra-content
+            POST_BUILD
+            WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
+            COMMAND cmake -E echo Skipping Extra Content)
 endif()

